cob_default_robot_config/cob4-2/mimic_configurations.yaml

```diff
@@ -0,0 +1 @@
+service_ns: /set_mimic
```

Comment:
this is a full service name. we should move mimic into a standalone namespace and the use this as ```service_ns```